<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure</string>
  </property>
  <widget class="QLabel" name="serverAddressLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Server Address: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverAddress">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="portBox">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>38</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>1024</number>
   </property>
   <property name="maximum">
    <number>65525</number>
   </property>
   <property name="value">
    <number>6000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="versionMessage">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
